torch/onnx/symbolic_opset9.py

```diff
@@ -1594,3 +1594,15 @@ def log2(g, self):
 
 def prim_shape(g, self):
     return g.op('Shape', self)
+
+
+@parse_args('v', 'i', 'v', 'v')
+def gather(g, self, dim, index, sparse_grad=False):
```

Comment:
This should also support Opset 10 export, correct?

Reply:
Yes, I think so. Should be able to fall back to opset 9's symoblics.